js-packages/@prestojs/rest/src/viewModelCachingMiddleware.ts

```diff
@@ -137,10 +146,38 @@ defaultGetDeleteId.validateEndpoint = (endpoint: Endpoint): void => {
  *
  * Each record instance created is also automatically added to the cache.
  *
+ * Usually when deleting a model, the endpoint doesn't return anything. So by default if the endpoint method is DELETE
+ * you should pass the model that will be deleted on a successful response.
+ *
+ * ```js
+ * const middleware = [viewModelCachingMiddleware(User)];
+ * // Response is empty:
+ * const userDelete = new ViewModelEndpoint(new UrlPattern('/users/:id/'), {
```

Comment:
oops - existing comments use `ViewModelEndpoint` but it's just `Endpoint` now. Could you replace all usages with `Endpoint`?

Reply:
done